clientd3d/intrface.h

```diff
@@ -40,13 +40,10 @@
 #define GRAPHICS_TOP_HEIGHT 24
 
 /* Inventory area */
-//#define INVENTORY_MIN_WIDTH (170 + LEFT_BORDER)
-#define INVENTORY_MIN_WIDTH (185 + LEFT_BORDER + 67)
-#define INVENTORY_MAX_WIDTH (250 + LEFT_BORDER)
+#define INVENTORY_MIN_WIDTH (215 + LEFT_BORDER + 67)
 
-//	MiniMap area.
-#define MINIMAP_MAX_WIDTH	( INVENTORY_MAX_WIDTH + 3 ) & ~3
-#define MINIMAP_MAX_HEIGHT	MINIMAP_MAX_WIDTH
+/* MiniMap area */
+#define MINIMAP_MAX	9000
```

Comment:
Name could be improved--MINIMAP_MAX_AREA?

Reply:
applied